deploy/kubedirector/kubedirector.hpe.com_kubedirectorapps_crd.yaml

```diff
@@ -63,6 +63,8 @@ spec:
                   pattern: '^(file|https?)://.+\.tgz$'
                 useNewSetupLayout:
                   type: boolean
+            defaultMaxLogSizeDump:
+              type: string
```

Comment:
I think these two new fields should be schema typed as integer.

Reply:
Hmm. Well for ASCII the distinction is moot, but yeah semantically we're talking about number of characters. So yes integer type would be a good choice, especially since we're dealing with small-ish values. (I don't know why I used 5M as an example above.)